stanford-sw/translation_maps/library_on_order_map_orig.properties

```diff
@@ -0,0 +1,34 @@
+# Map ON-ORDER Library Codes to User Friendly Description - Long Version for Facet Values
```

Comment:
so neither file existed before?  @jvine:  any reason to keep both versions in the code?   I *think* it's just a matter of how the locations percolate through indexing and UI code - when they get translated ... is that true, Laney?